pandas/util/testing.py

```diff
@@ -958,35 +980,52 @@ def assert_categorical_equal(left, right, check_dtype=True,
         Categoricals to compare
     check_dtype : bool, default True
         Check that integer dtype of the codes are the same
-    obj : str, default 'Categorical'
-        Specify object name being compared, internally used to show appropriate
+    obj : str or 2-length sequence (tuple, list, ...), default 'Categorical'
+        Specify object(s) name(s) being compared, used to show appropriate
         assertion message
     check_category_order : bool, default True
         Whether the order of the categories should be compared, which
         implies identical integer codes.  If False, only the resulting
         values are compared.  The ordered attribute is
         checked regardless.
     """
+
+    # obj = com._maybe_make_list(obj)
+
     _check_isinstance(left, right, Categorical)
 
+    obj = com._maybe_make_list(obj)
+
     if check_category_order:
         assert_index_equal(left.categories, right.categories,
-                           obj='{obj}.categories'.format(obj=obj))
+                           obj=map_obj('{obj}.categories', obj))
```

Comment:
might be easier to push the formatting down and pass this like
``obj=obj, message='categories'`` (not sure if this makes it easier to read / better)